Agent/SessionReplay/ViewCaptors/ViewDetails.swift

```diff
@@ -8,6 +8,7 @@
 
 import Foundation
 import UIKit
+import NewRelicPrivate
```

Comment:
This should be annotated `@_implementationOnly`, or in the future, as `internal`
```suggestion
import NewRelicPrivate
```
```suggestion
@_implementationOnly import NewRelicPrivate
```